.dockerdev/Dockerfile

```diff
@@ -36,12 +36,12 @@ RUN apt-get update -qq && DEBIAN_FRONTEND=noninteractive apt-get -yq dist-upgrad
     nodejs \
     $(cat /tmp/Aptfile | xargs) && \
     apt-get clean && \
+    npm install -g npm \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
-    truncate -s 0 /var/log/*log
+    truncate -s 0 /var/log/*log &&
 
 # Configure bundler
 ENV LANG=C.UTF-8 \
-  BUNDLE_JOBS=4 \
```

Comment:
I have 8 cores and since bundler now handles that automatically in newer versions, there is no need to limit me to 4 simultaneous downloads.